mobile/src/main.jsx

```diff
@@ -59,14 +52,14 @@ const renderAppWithPersistedState = persistedState => {
       isClientRegistered(client).then(clientIsRegistered => {
         if (clientIsRegistered) {
           const options = {
-            onError: onError(store, callback),
+            onError: onError(store.dispatch, store.getState, callback),
```

Comment:
There is no need to pass `callback` here as `callback` is used by `react-router` to validate an asyncrhonous check on routes. Cf. https://github.com/cozy/cozy-files-v3/pull/179/files#r106179418